src/powershell/Program.cs

```diff
@@ -327,6 +327,12 @@ private static int ExecPwshLogin(string[] args, string pwshPath, bool isMacOS)
                 return Exec("/bin/zsh", execArgs);
```

Comment:
@rjmholt, your last commit had 1 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/CHANGELOG/README.md.preview.md should work
```powershell
Tool reported URL as unreachable.
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 124
124:                             throw "Tool reported URL as unreachable."

```